src/_codux/boards/sort-by-button/sort-by-button.board.tsx

```diff
@@ -0,0 +1,8 @@
+import { createBoard } from '@wixc3/react-board';
+import SortByButton from '../../../components/SortByButton/SortByButton';
+
+export default createBoard({
+    name: 'SortByButton',
+    Board: () => <SortByButton />,
+    isSnippet: true,
```

Comment:
Currently, the button looks misplaced.
Since the size of this button is relative to its parent size, provide size to the canvas.
```
.sortByButton {
   width: 25%;
   height: 4em;
}
```